### ISSUE TITLE
Typo on doc

### ISSUE THREAD

**Issue by guglielmocamporese** [tensorflow/tensorflow#35136]
Line 194 is missing the square bracket:

- **wrong**
`Use distribution to create a linear combination of value with shape batch_size, Tq, dim]:`

- **correct**
`Use distribution to create a linear combination of value with shape [batch_size, Tq, dim]:`

Link to the line code:
https://github.com/tensorflow/tensorflow/blob/r2.0/tensorflow/python/keras/layers/dense_attention.py#L194

**Comment by yongtang** [comment]
@guglielmocamporese Are you willing to open a PR to fix the doc?
